### DISCUSSION TITLE
Certificate extension

### DISCUSSION THREAD

**Discussion by MuhammedKalkan** [MuhammedKalkan/OpenLens#173] [Announcements]
Hi all,

We had our signing certificate almost 1 year ago and it has a time limit of 1 year. In mid November it is gonna wear off.

So now once more we need funds to complete the price.
@ebbypeter are you willing to extend it and help community out once more? Maybe this time you can directly collect funds to buy the cert?

I got remaining 20 bucks. Anyone who wants to donate on top of it is welcome.

Update: Since the source repo is not active for some time, it is decided to wait and see until a new version is available. Thanks for your interest

**Comment by raphaelauv** [comment]
thanks for maintaining the repo :+1: 

how much is necessary ?

how can we give ?

thanks

**Comment by ebbypeter** [comment]
I am happy to renew it for another year, but if we need to do a community fund raising (if we need 2-3 year cert), that is good as well. 
I have a 50% off coupon code that I can apply.

The cost of renewals are as follows:
| Validity | Renewal Cost | Renewal Cost with Coupon |
|----------|--------------|--------------------------|
| 1 Year.  | $379.00      | $189.50                  |
| 2 Year.  | $644.00      | $322.00                  |
| 3 Year.  | $798.00      | $399.00                  | 

Let me know your thoughts.
Re: How can community contribute, @MuhammedKalkan maybe the best person to answer.

**Comment by MuhammedKalkan** [comment]
Unfortunately I don't have means to collect money anymore. @ebbypeter  or @jkroepke , you guys may help this out ?

**Reply by jkroepke** [reply]
Since https://github.com/lensapp/lens/ is dead and Lens Desktop (pro version) will now develop on private repository. I have the feeling that renew a certificate isn't worth anymore.

**Reply by ebbypeter** [reply]
The current signing cert expires on Nov 18th.
Does it mean that OpenLens is not installable after that?

**Reply by MuhammedKalkan** [reply]
It is true lens repo is not receiving any updates for a while now.  I dont expect them to make any major updates unless they really need it on their proprietary product. They claim that they did not abandon it , here: https://docs.k8slens.dev/faq/#q-there-is-a-lens-release-out-on-the-lens-forums-but-no-new-release-on-github-is-openlens-discontinuedabandoned

**Reply by jkroepke** [reply]
They may not abandon it, but we should not expect any updates there. Then, lets observe that the situation, but I would not recommend to buy a certificate for $180.

**Reply by adamhancock** [reply]
I don't think the certificate expiring will prevent existing versions from installing but we should prevent new releases (if any) from being automatically created if this is the route we go.

**Reply by MuhammedKalkan** [reply]
Alright then, best would be to observe situation and talk about certs if a new version comes along

**Reply by ebbypeter** [reply]
Ok... So, to be clear, I won't renew the cert unless we decide it otherwise. 
And we will stop creating new releases of OpenLens.

**Reply by MuhammedKalkan** [reply]
Correct

**Comment by jizi** [comment]
Hi, it seems there are some new releases present in https://github.com/lensapp/lens/releases. What needs to be done build new version here? Thx

**Reply by gdlx** [reply]
These are not new releases of OpenLens but closed source app contributions made through extensions.

https://github.com/lensapp/lens?tab=readme-ov-file#history-of-this-repository

At least they finally confirmed OpenLens isn't maintained anymore (and removed the discussion tab where they never answered about this...)
